api/handler.go

```diff
@@ -0,0 +1,43 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/hashicorp/consul-terraform-sync/api/oapigen"
+	"github.com/hashicorp/consul-terraform-sync/logging"
+)
+
+// Make sure we conform to ServerInterface
+var _ oapigen.ServerInterface = (*Handler)(nil)
+
+const (
+	handlerSubsystemName = "handler"
+)
+
+// Handler is composed of CTS server handlers which allows
+// the handler to adhere to the generated server interface
+type Handler struct {
```

Comment:
nit: what are your thoughts on naming `Handler` and its subsystem name to something like `Handlers` (plural)? It seems like `Handler` is an aggregation of the handlers rather than a handler itself but I might be misunderstanding.